Kirill_Lazniuk/Homework-12/HW-12.py

```diff
@@ -0,0 +1,17 @@
+import random
```

Comment:
up to you: you can only import the "randint"

```suggestion
from random import randint
```